test/ui/test.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// This is a UI test suite that makes sure various parts of the app look
+// as expected, and validates a subset of interactions
+
+const assert = require('assert');
+const selenium = require('selenium-webdriver'),
+      By = selenium.By,
+      until = selenium.until;
+const test = require('selenium-webdriver/testing');
+const fs = require('fs');
+const resemble = require('node-resemble');
+
+var driver = null;
+
+const suiteTimeout = 60000;
+const misMatchThreshold = 0;
+const goldenPathPrefix = 'ui/goldens/';
```

Comment:
I think I would prefer no 's' on this name. That also makes it consistent with the 'broken' directory name.
Making them consistent by naming the other one 'brokens' seems a lot worse.